duckduckgo/iOS
# Implement promo RMF style

Task/Issue URL: https://app.asana.com/0/0/1204954301997719/f
Tech Design URL:
CC:

**Description**:

Implements new RMF promo style and uses the new RemoteMessaging package.

Note that 'cta' parameter has been changed to 'message'.

Description text supports markdown and bold `<b></b>` HTML (on iOS 15+).

**Steps to test this PR**:
1. Clean install and get through onboarding.
2. New card should display - copy is for example only
3. Check pixels fire when share sheet has finished
4. Check pixel fires for new `action` type
5. Smoke test RMF



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
